addons/mail/models/mail_message.py

```diff
@@ -1125,15 +1131,20 @@ def _message_format_personalize(self, partner_id, messages_formatted=None, forma
         for vals in messages_formatted:
             # set value for user being a follower, fallback to False if not prepared
             follower_id_by_pid = vals.pop('follower_id_by_partner_id', {})
-            vals['user_follower_id'] = follower_id_by_pid.get(partner_id, False)
+            follower_id = follower_id_by_pid.get(partner_id, False)
+            if follower_id:
+                vals['originThread']['selfFollower'] = {
+                    'id': follower_id,
+                    'is_active': True,
+                    'partner': {'id': partner_id, 'type': "partner"},
+                }
         return messages_formatted
 
     def _get_message_format_fields(self):
         return [
             'id', 'body', 'date', 'email_from',  # base message fields
             'message_type', 'subtype_id', 'subject',  # message specific
-            'model', 'res_id', 'record_name',  # document related
-            'starred_partner_ids',  # list of partner ids for whom the message is starred
+            'model', 'res_id', 'record_name',  # document related FIXME need to be kept for mobile app as iOS app cannot be updated
```

Comment:
I would not keep that support unless it's actually proven to be necessary.

Reply:
`message_format` makes use of them (`val['model']`, etc.).
If we use `message_sudo.model`, I'm afraid it might increase query counts similarly to what `starredPersonas`